components/Header/index.js

```diff
@@ -3,11 +3,11 @@ import Link from 'next/link';
 
 const Header = (props) => {
   return (
-    <Link href="/">
-      <StyledHeader {...props}>
-        <LogoImg src={'/images/icons/votu_logo.svg'} />
-      </StyledHeader>
-    </Link>
+    <StyledHeader {...props}>
+      <Link href="/">
+        <LogoImg src={'/images/icons/votu_pe_logo.svg'} />
```

Comment:
```suggestion
        <LogoImg src="/images/icons/votu_pe_logo.svg" />
```
Es raro creo que es el único asset que tiene `_` en su nombre. Podemos aprovechar en actualizarlo a `-`?

Reply:
Hecho, agregué un commit https://github.com/openpolitica/open-wizard/pull/259/commits/8feebbac90e548b9a9fb0748b4b2f39f529836c1